packages/web/src/components/tiktok-button/TikTokButton.tsx

```diff
@@ -0,0 +1,35 @@
+import {
+  Button,
+  ButtonSize,
+  ButtonType,
+  IconTikTokInverted
+} from '@audius/stems'
+import cn from 'classnames'
+
+import styles from './TikTokButton.module.css'
+
+type TikTokButtonProps = {
+  text?: string
+  textClassName?: string
+  iconClassName?: string
+  onClick?: () => void
+  className?: string
+}
+
+export const TikTokButton = (props: TikTokButtonProps) => {
+  const { text, className, onClick, textClassName, iconClassName } = props
+  return (
+    <Button
+      type={ButtonType.PRIMARY_ALT}
+      leftIcon={<IconTikTokInverted className={styles.icon} />}
+      className={cn(styles.button, styles.includeHoverAnimations, {
+        [className!]: !!className
```

Comment:
Do we just have className as an option in cs? If it's falsy it won't be included

Reply:
Not sure what you mean by `cs`, this is mostly copy and paste from the twitter and ig buttons